index.js

```diff
@@ -65,30 +67,88 @@ function deepmergeConstructor (options) {
     return isMergeableObject(entry)
       ? Array.isArray(entry)
         ? cloneArray(entry)
-        : mergeObject({}, entry)
+        : cloneObject(entry)
       : entry
   }
 
-  function mergeObject (target, source) {
+  function mergeObjectNullPrototype (target, source) {
+    const result = Object.create(null)
+    const targetKeys = getKeys(target)
+    const sourceKeys = getKeys(source)
+    let i, il, key
+    for (i = 0, il = targetKeys.length; i < il; ++i) {
+      (key = targetKeys[i], true) &&
+        (sourceKeys.indexOf(key) === -1) &&
+        (result[key] = clone(target[key]))
```

Comment:
Is this syntax part of the speed improvements?

Reply:
yes